common/services/app/civic-uk/scripts.tsx

```diff
@@ -1,3 +1,38 @@
+import { font } from '@weco/common/utils/classnames';
+
+const branding = {
+  removeIcon: true,
+  removeAbout: true,
+  fontFamily: 'Inter, sans-serif',
+};
+
+const text = {
+  title: 'Manage cookies',
+  intro: '',
+  necessaryTitle: 'Essential cookies',
+  necessaryDescription:
+    'These cookies are necessary for our website to function and therefore always need to be on.',
+  notifyTitle: 'Our website uses cookies',
+  notifyDescription:
+    "We use cookies to make our website work. To help us make our marketing and website better, we'd like your consent to use cookies on behalf of third parties too.",
+  thirdPartyTitle: 'HI',
+  closeLabel: 'Close preference centre',
+  settings: 'Manage cookies',
+  accept: 'Accept all',
+  reject: 'Essential only',
+};
+
+// Should your privacy policy change after a user gives consent,
+// Cookie Control will invalidate prior records of consent and seek the user's preferences using the latest information available.
+// https://www.civicuk.com/cookie-control/documentation/cookies
+const statement = {
+  description:
+    "We use cookies to make our website work. To help us make our marketing and website better, we'd like your consent to use cookies on behalf of third parties too.",
+  name: 'Find out more in our privacy policy',
+  url: 'https://wellcome.org/who-we-are/privacy-and-terms',
+  updated: '25/05/2018',
+};
```

Comment:
Config to be found here: https://www.civicuk.com/cookie-control/documentation/cookies#privacy-statement

Would be good to have a date on the page as well I reckon, and to know what date that is for our config; I don't think it can be found at the mo on the page?

Reply:
Not directly related to this PR, but should we have our own privacy statement page on `wellcomecollection.org`? We'll have different cookies from `wellcome.org` and being able to manage that list ourself would be useful.

Reply:
I think I remember we did say it would be the way we'd go and in that case, we want to make sure we have a date there (that I think will just be part of the copy as it shouldn't change based on publication if someone republishes to fix a typo). Or I wonder if we could fetch that date from ?somewhere? and display it in both places automatically, that'd be ideal.